index.html

```diff
@@ -69,7 +69,10 @@
 <h3>
 <a id="support-or-contact" class="anchor" href="#support-or-contact" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Support or Contact</h3>
 
-<p>You can contact us and report issues on <a href="https://github.com/spotbugs/spotbugs/issues">GitHub</a></p>
+<p>You can report issues on <a href="https://github.com/spotbugs/spotbugs/issues">GitHub</a>.</p>
+          
+<p>Or, you can contact us using <a href="https://github.com/spotbugs/discuss/issues?q=">our general purpose mailing list</a>.</p>
```

Comment:
What should "?q=" do useful? Shouldn't we just use "https://github.com/spotbugs/discuss/issues"?